tests/playwright/src/specs/kubernetes.spec.ts

```diff
@@ -106,15 +109,15 @@ test.describe('Kubernetes resources End-to-End test', () => {
       const pvcDetails = await pvcsPage.openResourceDetails(PVC_NAME, KubernetesResources.PVCs);
       await playExpect(pvcDetails.heading).toBeVisible();
       await playExpect
-        .poll(async () => pvcDetails.getState(), { timeout: 40_000 })
+        .poll(async () => pvcDetails.getState(), { timeout: 50_000 })
         .toEqual(KubernetesResourceState.Starting);
     });
     test('Bind the PVC to a pod', async ({ navigationBar }) => {
       const podsPage = await navigationBar.openPods();
       await playExpect(podsPage.heading).toBeVisible();
       const playYamlPage = await podsPage.openPlayKubeYaml();
       await playExpect(playYamlPage.heading).toBeVisible();
-      const yamlFilePath = path.resolve(__dirname, '..', '..', 'resources', 'kubernetes', `${POD_NAME}.yaml`);
+      const yamlFilePath = path.resolve(__dirname, '..', '..', 'resources', 'kubernetes', `${PVC_POD_NAME}.yaml`);
```

Comment:
please also move constant into file defition out of method. to have a little less code in the test methods.